fhir-server/liberty-config/server.xml

```diff
@@ -190,23 +190,6 @@
     </dataSource>
     -->
 
-    <dataSource id="fhirbatchDS" jndiName="jdbc/fhirbatchDB">
```

Comment:
Why can't the fhirbatchDS exist here? I suggest we comment it out, and remove server-withDb2JavaBatch.xml

Reply:
actually - I suggest

` <include optional="true" location="${server.config.dir}/batchDs.xml"/>`

Reply:
haha, yeah, sound good

Reply:
done